package.json

```diff
@@ -92,6 +93,7 @@
   ],
   "license": "MIT",
   "dependencies": {
+    "@babel/runtime": "7.0.0-rc.1",
```

Comment:
I haven’t used babel runtime since docs are very loose and end effect on final app is not clear.

The fact lib bundle gets smaller is good of course. But does it mean that babel runtime will be (has to be) added to an app?
When you have build in place that is not a big deal, but in case of static sites using minified prebuilt versions of libs it means no tree shaking. Full babel runtime is huge :(

Please correct me if I am wrong here (I wish I am)

Reply:
It's direct dependency, so no affect to user. `@babel/plugin-transform-runtime` imports only necessary stuff. Here's what imported now
```js
import _extends from '@babel/runtime/helpers/esm/extends';
import _inheritsLoose from '@babel/runtime/helpers/esm/inheritsLoose';
```

it's exactly the same what `rollup-plugin-babel` inlines to the bundle without runtime plugin, so their's not gain or loss in size. The idea is that these helpers may be reused across many packages.

Actually the new babel runtime package is small. It was decoupled from core-js version. So even less stuff in user node_modules.

Reply:
Thanks for the explanation!